spec/services/greet_ungreeted_students_spec.rb

```diff
@@ -20,7 +20,8 @@
     # New section for the welcome message
     expect_any_instance_of(WikiEdits).to receive(:add_new_section).and_call_original
     # Templates are added to the top of user page, talk page, sandbox
-    expect_any_instance_of(WikiEdits).to receive(:add_to_page_top).thrice.and_call_original
+    expect(AddSandboxTemplate).to receive(:new)
```

Comment:
Does this still pass if you use `.and_call_original`? If so, that would be better. This test stubs the requests themselves, so it's designed for all the code up to the actual network requests to run.

Reply:
Nope, it fails when I use `.and_call_original`, it is having problems with the new `WikiEdits` instance I make there.